File: DotChat/DotChat/Messages/MessageType.cs
namespace K1vs.DotChat.Messages
{
    using System;

    [Flags]
    public enum MessageType
    {
        None = 0,
        Text = 1 << 0,
        Quote = 1 << 1,
        Attachment = 1 << 2,
        ChatRef = 1 << 3,
        Contact = 1 << 4,
        Custom = 1 << 16
    }
}

File: DotChat/Clients/DotChat.Clients.JavaScript/src/enums/message-type.js
export const MessageType = {
    none: 0,
    text: 1 << 0,
    quote: 1 << 1,
    attachment: 1 << 2,
    chatRef: 1 << 3,
    contact: 1 << 4,
    custom: 1 << 16
}